openml/extensions/sklearn/extension.py

```diff
@@ -777,17 +797,20 @@ def _get_external_version_string(
         # version of all subcomponents, which themselves already contain all
         # requirements for their subcomponents. The external version string is a
         # sorted concatenation of all modules which are present in this run.
-        model_package_name = model.__module__.split(".")[0]
-        module = importlib.import_module(model_package_name)
-        model_package_version_number = module.__version__  # type: ignore
-        external_version = self._format_external_version(
-            model_package_name, model_package_version_number,
-        )
-        openml_version = self._format_external_version("openml", openml.__version__)
-        sklearn_version = self._format_external_version("sklearn", sklearn.__version__)
 
         external_versions = set()
-        external_versions.add(external_version)
+
+        if model is not None:
```

Comment:
When is the model `None`?

Reply:
The model is None if the flow is a placeholder flow such as 'passthrough' or 'drop'. I'll add this to the docs.